traffic_portal/app/src/common/models/PropertiesModel.js

```diff
@@ -21,6 +21,7 @@ var PropertiesModel = function() {
 	this.loaded = false;
 
 	this.setProperties = function(properties) {
+		if (this.loaded == true) return;
```

Comment:
comparisons should use `===` - but also you don't need to compare booleans to constant values. `if (this.loaded)` works just fine.

Reply:
oh right. i like the `(this.loaded)` syntax